views/pulsar/v2/helpers/tabs.html.twig

```diff
@@ -74,8 +73,7 @@
                         'label': tab.label,
                         'href': href,
                         'data-toggle': type,
-                        'disabled': tab.disabled|default(false),
-                        'tabindex': tabindex
+                        'disabled': tab.disabled|default(false)
```

Comment:
Doesn't this potentially give the link an invalid `disabled` attribute?

Reply:
No, this `options` object is passed through the attribute parser which knows we're working with a link here, so uses `aria-disabled`.